source/crud/tests/v2/unacknowledged-bulkWrite-update-hint-clientError.yml

```diff
@@ -0,0 +1,94 @@
+runOn:
+  # These tests assert that the driver raises a client-side error when the hint
+  # option is passed with an unacknowledged update operation against a server
+  # that does not support hints in update operations.
+  - { maxServerVersion: "4.1.9" }
```

Comment:
Thanks for adding these missing tests too!

Reply:
No problem!